src/esy/buildEjectCommand/index.js

```diff
@@ -384,30 +384,14 @@ function buildEjectCommand(
     filename: ['bin/replace-string'],
     executable: true,
     contents: outdent`
-      #!/usr/bin/env python2
+      #!/usr/bin/env bash
 
-      import sys
-      import os
-      import stat
+      FILENAME="$1"
```

Comment:
The reason I initially did this is because of a vague recollection of this blog post: http://www.oilshell.org/blog/2016/11/06.html